app/build.gradle.kts

```diff
@@ -80,6 +78,20 @@ android {
         }
     }
     namespace = "com.google.samples.apps.nowinandroid"
+
+    baselineProfile {
```

Comment:
This should be outside the android component.

You can refer to the example in [this integration test](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:benchmark/integration-tests/baselineprofile-flavors-consumer/build.gradle;l=67).